gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Maven version
-version = 3.1.0-SNAPSHOT
```

Comment:
We not need to manage this version.

Reply:
Can we confirm all java-sdk changes compatible with android-sdk?

Reply:
Updated and tested with android sdk